src/cli/info.rs

```diff
@@ -0,0 +1,119 @@
+use structopt::StructOpt;
+
+use std::fmt;
+
+use termion::color;
+
+use crate::api;
+use crate::errors::*;
+
+/// Shows database information
+#[derive(Debug, StructOpt)]
+pub struct InfoOptions {
+    /// Shows statistics
+    #[structopt(short, long)]
+    stats: bool,
+
+    /// Shows tag usage breakdown
+    #[structopt(short, long)]
+    usage: bool,
+}
+
+impl InfoOptions {
+    pub fn execute(&self, global_opts: &super::GlobalOptions) -> Result<()> {
+        let db_path = super::locate_db(&global_opts.database)?;
```

Comment:
Methods from the parent module can be called even if they are not marked as `pub`. I used this property in a few places to make utility functions visible only to submodules.